### PR TITLE
Upgrade homepage

### PR DESCRIPTION
**Pull request by chartgerink**
This PR tracks the work on upgrading the homepage. This is a substantial change from what was there - most of the content from the existing homepage has moved into the new "Getting Started" page temporarily.

Timeline for this PR is:
* Ready for review June 6th
* Feedback closes June 23rd
* Integrate feedback (where possible) and merge by June 27th
 
Biggest open questions before merging:

- [x] If we keep the animated terminal recording, what do we show in this? Currently it runs a basic command and is not to be considered final. PS: This is being recorded and animated using [`terminalizer`](https://github.com/faressoft/terminalizer)
- [x] Do we have sign off to use the logos of our partners? I don't have a full sense of whether this is part of existing commitments/agreements
- [x] Are there fundamental objections to the copy? Additions, changes, refinements are easily made in the future and I want to now ensure nothing is objected to.

> [!NOTE]
> You can view the preview deployment on Netlify: https://deploy-preview-262--tourmaline-marshmallow-241b40.netlify.app/

I did my best for this assignment, as there were only minimal copy guidelines, scope definition, or needs for the homepage that I could base myself off of. As such, I am very happy to receive concrete, actionable, and proportional feedback. Once we merge this, that is only the starting point for this new homepage - we can keep working on it over time. So the question here is - "Do we want this to be the replacement for the existing homepage?"

---

After merging, the "Getting Started" will need further fleshing out. I would like to keep this outside the scope of this PR, and also understand that this is an important bit because we're linking to it. I will take suggestions for how to do this over in #272.

### CODE REVIEW

File: _components/Hero.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            </p>
+            <div className="tw-mt-10 tw-flex tw-items-center tw-justify-center tw-gap-x-6">
+              <a
+                href="./getting-started.html"
+                className="tw-rounded-md tw-bg-lime-500 tw-px-3.5 tw-py-2.5 tw-text-sm tw-font-semibold tw-text-black tw-shadow-sm hover:tw-bg-lime-400 focus-visible:tw-outline focus-visible:tw-outline-2 focus-visible:tw-outline-offset-2 focus-visible:tw-outline-lime-400"
+              >
+                Get started
+              </a>
+              <a
+                href="https://join.slack.com/t/epiversetrace/shared_invite/zt-2dl52uahe-vH1AtHgYUV_IptVk84Tzrw"
```
Comment (avallecam) [inline _components/Hero.jsx:36]: From previous discussions, we may want to redirect users to our [Discussions site on GitHub](https://github.com/orgs/epiverse-trace/discussions). This aims to keep the conversations fully open.

```suggestion
                href="https://github.com/orgs/epiverse-trace/discussions"
```

File: _components/Hero.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                className="tw-text-sm tw-font-semibold tw-leading-6 tw-text-white"
+              >
+                Join our chat <span aria-hidden="true">→</span>
```
Comment (avallecam) [inline _components/Hero.jsx:39]: Does this sound appropriate?

```suggestion
                Join our discussions <span aria-hidden="true">→</span>
```

**Review by avallecam — COMMENTED** [review]
Thanks for arranging the main website! Looks really great! 

I made one comment about the link to the chat, redirecting people to Discussions instead of Slack, 

Happy to read others input on this!

**Comment by joshwlambert** [comment]
@chartgerink really nice work revamping the website. I prefer the new layout. I like the spacing between information, it doesn't feel cluttered.

> If we keep the animated terminal recording, what do we show in this? Currently it runs a basic command and is not to be considered final

I'm not sure what should be shown. I think the current example is good. Another option could be to find a quick example that chains together a few functions from different Epiverse packages to highlight interoperability of our tools, but nothing comes to mind right now that would be easy to show in an equivalent example.

> Do we want this to be the replacement for the existing homepage?

Yes!

Other comments:

- The image next to "Better prepared, together" on the home page is cut off to the right. Could this be reformatted to include the whole image?

**Comment by adamkucharski** [comment]
Thanks for putting this together, it's looking much more polished. A few comments:

- In the past year, we’ve moved away from having epiverse.github.io as page just for developers, as we also needed something for potential users/leaners and the main epiverse.org page doesn’t have enough detail/actionable information for this audience. Probably doesn’t need much changing in practice, just worth being aware of who will be landing here when designing copy etc.

- I don’t think having an R terminal animation is best use of visual space for a landing page - expect most visitors will be interested in what tools can do, so like other package pages (tidyverse etc.) I’d suggest we showcase key functionality in a space-efficient way instead. Maybe a carousel cycling through useful plots/outputs from a few packages? Have lots of illustrative examples from talks if useful (some [here](https://epiverse-trace.github.io/slides/juniper-may-2023/)).

- Suggest expand main subtitle applications to something like “From reading and cleaning your data to estimating vaccine efficacy and policy impact, …” 

- Alongside new package development, we want to promote visible integration with the wider ecosystem and contributions to existing packages (e.g. EpiNow2, epireview, godataR, incidence2, socialmixr...). So would be nice to highlight these modular links in copy, to avoid the risk that people interpret Epiverse-TRACE tools as a stand-alone option.

- Current roadmap page on 'get started' is focused on LSHTM-MRCG tools, so we should think about expanding to include TRACE-LAC tools (this has been on to do list for a while – also links to issue #272 ). 

- Would suggest moving partners to bottom of page (and maybe include phase 2 partners as well, as they are announced?) as this means calls to action (like ‘want to help’) can be higher on page for user.

- Summit image formatting seemed off for me on mobile. Could also include carousel from other events? (e.g. WHO Hub Collaboratory). But probably not a priority for this first version.

- 'Join the chat' points to Slack space, but as this is non-public & paid, would link to GitHub discussion board be better?

Related issues:
- I noticed the R package link on main [Epiverse page](https://data.org/initiatives/epiverse/) goes to R Universe, which is not best landing page, so we should redirect to this new page.
- We actually have epiverse-trace.org and epiverse.io previously registered for some testing purposes, so could use one of these for new site when ready.

**Comment by chartgerink** [comment]
Thanks all for the feedback! I incorporated a bunch of things for the homepage that focused on existing content. It is out of scope for me at this time to swap out entire components for new ones.

**Review by Bisaloo — APPROVED** [review]

**Pull request merged**